### PR TITLE
I877032: Added RabbitManagementApi tests

### PR DESCRIPTION
**Pull request by kusumaghoshdastidar**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=877032

### CODE REVIEW

File: autoscale-workload-rabbit/pom.xml [hunk 1]
```diff
@@ -52,6 +52,16 @@
...
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-jackson</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```
Comment (dermot-hardy) [inline autoscale-workload-rabbit/pom.xml:64]: Why would the service have two extra runtime dependencies just to add some tests?

Comment (kusumaghoshdastidar) [reply autoscale-workload-rabbit/pom.xml:64]: These were missing from the previous PR which I found when I added the tests.

Comment (dermot-hardy) [reply autoscale-workload-rabbit/pom.xml:64]: So these are actually required for it to work and it is not actually fixed as things stand?

Comment (kusumaghoshdastidar) [reply autoscale-workload-rabbit/pom.xml:64]: Yes

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**